### ISSUE TITLE
Add relevant links in the footer

### ISSUE THREAD

**Issue by t0xic0der** [fedora-infra/mote#170]
Like the one at https://getfedora.org/.

![Screenshot from 2021-08-30 21-04-08](https://user-images.githubusercontent.com/49605954/131365114-84e0de41-e864-4d49-b3ee-9fea259baaf3.png)

**Comment by siddhantkhariwal** [comment]
Hey, Can I fix this issue?

**Comment by t0xic0der** [comment]
Sure, please go ahead.

**Comment by t0xic0der** [comment]
@siddhantkhariwal, could you please provide an ETA for this?

**Comment by siddhantkhariwal** [comment]
Will complete in 24hrs.

**Comment by t0xic0der** [comment]
Sounds good. Please derive a new branch from the `fragment` branch https://github.com/fedora-infra/mote/tree/fragment and be sure to target your PR against it.

**Comment by siddhantkhariwal** [comment]
> Sounds good. Please derive a new branch from the `fragment` branch https://github.com/fedora-infra/mote/tree/fragment and be sure to target your PR against it.

When I am opening this link it is showing an error 404 page.

**Comment by t0xic0der** [comment]
We moved that branch to become the primary branch for the repository. Now, you can derive from the `main` branch and target the `main` branch in your PR.

**Comment by siddhantkhariwal** [comment]
> We moved that branch to become the primary branch for the repository. Now, you can derive from the `main` branch and target the `main` branch in your PR.

Okay, Thank you.

**Comment by siddhantkhariwal** [comment]
> We moved that branch to become the primary branch for the repository. Now, you can derive from the `main` branch and target the `main` branch in your PR.

Hello @t0xic0der . I have made the footer with all the relevant links but I am having trouble with the Facebook and Twitter icons. It is showing a blank square instead of the original icon. I searched on stack overflow but was still not able to get the icons. Other than that the footer is complete.

**Comment by siddhantkhariwal** [comment]
Hey @t0xic0der can you help me with this?

> > We moved that branch to become the primary branch for the repository. Now, you can derive from the `main` branch and target the `main` branch in your PR.
> 
> Hello @t0xic0der . I have made the footer with all the relevant links but I am having trouble with the Facebook and Twitter icons. It is showing a blank square instead of the original icon. I searched on stack overflow but was still not able to get the icons. Other than that the footer is complete.

**Comment by t0xic0der** [comment]
Sure. Can you check if you can obtain the icons from FontAwesome?

**Comment by siddhantkhariwal** [comment]
> Sure. Can you check if you can obtain the icons from FontAwesome?

Yes, I tried with font awesome icons but it is showing only blank squares, I am unable to solve that issue.

**Comment by t0xic0der** [comment]
@siddhantkhariwal,

Could you make it happen without those icons? We can always add those icons in a subsequent PR.

**Comment by siddhantkhariwal** [comment]
> @siddhantkhariwal,
> 
> Could you make it happen without those icons? We can always add those icons in a subsequent PR.

Yeah, okay I'll just create a pr.

**Comment by siddhantkhariwal** [comment]
Hey, @t0xic0der I created a pr, kindly check through it and let me know for any feedback.

**Comment by subhangi2731** [comment]
@t0xic0der @darknao I want to work on this issue. please assign me this one

**Comment by yashbajoria05** [comment]
Please this issue to me

**Comment by RuthM-K** [comment]
Hello @gridhead, Can I work on this issue as well?

**Comment by RuthM-K** [comment]
Hello @gridhead I'm also an Outreachy applicant; Can i be assigned this issue please?

**Comment by gridhead** [comment]
We already have a pending pull request on this issue ticket https://github.com/fedora-infra/mote/pull/698.

Please feel free to collaborate with the author of the pull request.
